### PR TITLE
[R2] Note clarifying what can be in a bucket name

### PR DESCRIPTION
**Pull request by crwaters16**
Expanding on feedback from https://github.com/cloudflare/cloudflare-docs/pull/13345

### CODE REVIEW

File: content/r2/buckets/create-buckets.md [hunk 1]
```diff
@@ -21,8 +21,13 @@ The R2 support in Wrangler allows you to manage buckets and perform basic operat
 Create a bucket:
 
 ```sh
-$ wrangler r2 bucket create YOUR_BUCKET_NAME
+$ wrangler r2 bucket create YOUR-BUCKET-NAME
```
Comment (elithrar) [inline content/r2/buckets/create-buckets.md]:
```suggestion
$ wrangler r2 bucket create your-bucket-name-here
```

Comment (elithrar) [reply content/r2/buckets/create-buckets.md]: Make the placeholder name compliant too?

File: content/r2/buckets/create-buckets.md [hunk 2]
```diff
@@ -21,8 +21,13 @@ The R2 support in Wrangler allows you to manage buckets and perform basic operat
 Create a bucket:
 
 ```sh
-$ wrangler r2 bucket create YOUR_BUCKET_NAME
+$ wrangler r2 bucket create your-bucket-name-here
 ```
+{{<Aside type="note" header="Header text">}}
```
Comment (patriciasantaana) [inline content/r2/buckets/create-buckets.md]: Should "Header text" be something else?
```suggestion
{{<Aside type="note" header="Header text">}}
```

**Review by patriciasantaana — APPROVED** [review]

File: content/r2/buckets/create-buckets.md (hunk 2 above)
Comment (crwaters16) [inline content/r2/buckets/create-buckets.md]:
```suggestion
{{<Aside type="note">}}
```

File: content/r2/buckets/create-buckets.md [hunk 3]
```diff
@@ -21,8 +21,13 @@ The R2 support in Wrangler allows you to manage buckets and perform basic operat
 Create a bucket:
 
 ```sh
-$ wrangler r2 bucket create YOUR_BUCKET_NAME
+$ wrangler r2 bucket create your-bucket-name-here
```
Comment (crwaters16) [inline content/r2/buckets/create-buckets.md]:
```suggestion
$ wrangler r2 bucket create your-bucket-name
```

**Pull request merged**